src/main/java/com/avispl/symphony/dal/communicator/aggregator/common/GoogleWorkspaceConstant.java

```diff
@@ -64,4 +65,7 @@ public class GoogleWorkspaceConstant {
 	public static final String HOUR = " h ";
 	public static final int MAC_ADDRESS_LENGTH = 12;
 	public static final int DEFAULT_ORG_UNIT_POSITION = 0;
+	public static final int MIN_VOLUME_LEVEL = 0;
+	public static final String REGEX_SERIAL_NUMBER = "^[a-zA-Z0-9]+$";
+	public static final String CPU_TEMPERATURE = "Core1(C)";
```

Comment:
Is it always gonna be "Core1"? Wonder if we need 1 then

Reply:
> Is it always gonna be "Core1"? Wonder if we need 1 then

I renamed it "CPU Temperature", it includes data of all Core